### PR TITLE
Adding a sitemap to 11ty site

### PR DESCRIPTION
**Pull request by Akbarsait**
Hi, 
Submitting the PR for adding a sitempa to 11ty site.

### CODE REVIEW

**Comment by peruvianidol** [comment]
This is great. One thing though, you'll need to escape the Nunjucks using `{% raw %}{%endraw%}` within the code sample so that the Nunjucks isn't compiled into the page. Thanks!

File: _src/recipes/010-add-a-sitemap.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Add a Sitemap to 11ty site
+authorName: Akbarsait
+authorUrl: https://www.akbarsait.com
+---
+
+Sitemap helps search engine crawl for making easier and quicker way to find and index your website content. The base blog of 11ty is already equipped but here we have added few changes especially for the dates to make it easier and readable. 
+
+### Directions
+1. Create a file in your source directory  called **sitempa.njk** and add the following content to it.
```
Comment (peruvianidol) [inline _src/recipes/010-add-a-sitemap.md]: Also you have a typo here in the name of the file.

File: _src/recipes/010-add-a-sitemap.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```md
+---
+permalink: /sitemap.xml
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    {% for page in collections.all %}
+        <url>
+            <loc>{{ site.url }}{{ page.url | url }}</loc>
```
Comment (peruvianidol) [inline _src/recipes/010-add-a-sitemap.md:31]: This assumes the user has a `site.json` file, which we don't have a recipe for yet. You can either make a recipe and add it to the top as a prerequisite, or just note which values need to exist in data.json for this to work.

File: _src/recipes/010-add-a-sitemap.md (hunk 1 above)
Comment (Akbarsait) [reply _src/recipes/010-add-a-sitemap.md]: Thanks. I have updated it

File: _src/recipes/010-add-a-sitemap.md (hunk 2 above)
Comment (Akbarsait) [reply _src/recipes/010-add-a-sitemap.md:31]: I have updated and pushed the changes now.

File: _src/recipes/010-add-a-sitemap.md [hunk 3]
```diff
@@ -16,20 +26,20 @@ eleventyExcludeFromCollections: true
 ---
 <?xml version="1.0" encoding="utf-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-    {% for page in collections.all %}
+    {% raw %}{% for page in collections.all %}
         <url>
             <loc>{{ site.url }}{{ page.url | url }}</loc>
             <lastmod>{{ page.date.toISOString() }}</lastmod>
             <changefreq>{{page.data.changeFreq}}</changefreq>
```
Comment (peruvianidol) [inline _src/recipes/010-add-a-sitemap.md]: Where does `page.data.changeFreq` come from? Is that supplied by Eleventy or does it need to be included in the front matter? Is this value optional?

Comment (Akbarsait) [reply _src/recipes/010-add-a-sitemap.md]: we have handle it through front matter data. Considering the simplicity of the recipe, i'm going to take it off now and commit the update.

**Review by Akbarsait — COMMENTED** [review]
i have commented out the to avoid further confusion and committed a change for review.

**Comment by Akbarsait** [comment]
Is there anything pending from my end? Kindly confirm

**Comment by peruvianidol** [comment]
You're good. Thanks. I'll merge it as soon as I have some time.

**Pull request merged**